docs/ref/csrf.txt

```diff
@@ -146,6 +148,17 @@ If you're using AngularJS 1.1.3 and newer, it's sufficient to configure the
     $httpProvider.defaults.xsrfCookieName = 'csrftoken';
     $httpProvider.defaults.xsrfHeaderName = 'X-CSRFToken';
 
+CSRF token in Jinja2 templates
+------------------------------
+
+Django's Jinja2 template engine exposes ``{{ csrf_input }}`` which is
+equivalent to ``{% csrf_token %}`` in the Django template language. For
+example:
+
+.. code-block:: html
+
+    <form action="" method="post">{{ csrf_input }}
+
 Other template engines
 ----------------------
 
```

Comment:
On further review, I think the (unmodified) following paragraph really needs some updates, too. E.g instead of "When using a different template engine than Django's built-in engine," it should say "When using a different template engine than one of the built-in engines," and it really shouldn't use Jinja as an example anymore, since we've just shown a better way to do it in Jinja.

Also, I'm not sure why it recommends using Javascript to get the token value; if you're rendering a template on the server side that recommendation makes no sense. So that part should be updated to show how to correctly get the token value on the server side, from the request, the same way the Jinja2 backend does.

Or, on the other hand, the population of people using Django with a template engine other than DTL or Jinja2 is probably small enough that we could maybe just delete the whole following section rather than try to figure out how to update it (and what obscure templating language to use in an updated example).